cs-hashicorp-terraform/src/main/java/io/cloudslang/content/hashicorp/terraform/actions/runs/ApplyRun.java

```diff
@@ -0,0 +1,155 @@
+                                       @Param(value = EXECUTION_TIMEOUT, description = EXECUTION_TIMEOUT_DESC) String executionTimeout,
+                                       @Param(value = ASYNC, description = ASYN_DESC) String asyn,
+                                       @Param(value = POLLING_INTERVAL, description = POLLING_INTERVAL_DESC) String pollingInterval,
+                                       @Param(value = KEEP_ALIVE, description = KEEP_ALIVE_DESC) String keepAlive,
+                                       @Param(value = CONNECTIONS_MAX_PER_ROUTE, description = CONN_MAX_ROUTE_DESC) String connectionsMaxPerRoute,
+                                       @Param(value = CONNECTIONS_MAX_TOTAL, description = CONN_MAX_TOTAL_DESC) String connectionsMaxTotal,
+                                       @Param(value = RESPONSE_CHARACTER_SET, description = RESPONSE_CHARACTER_SET_DESC) String responseCharacterSet) {
+
+
+        runId = defaultIfEmpty(runId, EMPTY);
+        runComment = defaultIfEmpty(runComment, EMPTY);
+        requestBody = defaultIfEmpty(requestBody, EMPTY);
+        proxyHost = defaultIfEmpty(proxyHost, EMPTY);
+        proxyPort = defaultIfEmpty(proxyPort, DEFAULT_PROXY_PORT);
+        proxyUsername = defaultIfEmpty(proxyUsername, EMPTY);
+        proxyPassword = defaultIfEmpty(proxyPassword, EMPTY);
+        trustAllRoots = defaultIfEmpty(trustAllRoots, BOOLEAN_FALSE);
+        x509HostnameVerifier = defaultIfEmpty(x509HostnameVerifier, STRICT);
+        trustKeystore = defaultIfEmpty(trustKeystore, DEFAULT_JAVA_KEYSTORE);
+        trustPassword = defaultIfEmpty(trustPassword, CHANGEIT);
+        connectTimeout = defaultIfEmpty(connectTimeout, CONNECT_TIMEOUT_CONST);
+        socketTimeout = defaultIfEmpty(socketTimeout, ZERO);
+        executionTimeout = defaultIfEmpty(executionTimeout, EXEC_TIMEOUT);
+        asyn = defaultString(asyn, BOOLEAN_FALSE);
+        pollingInterval = defaultString(pollingInterval, POLLING_INTERVAL_DEFAULT);
+        keepAlive = defaultIfEmpty(keepAlive, BOOLEAN_TRUE);
+        connectionsMaxPerRoute = defaultIfEmpty(connectionsMaxPerRoute, CONNECTIONS_MAX_PER_ROUTE_CONST);
+        connectionsMaxTotal = defaultIfEmpty(connectionsMaxTotal, CONNECTIONS_MAX_TOTAL_CONST);
+        responseCharacterSet = defaultIfEmpty(responseCharacterSet, UTF8);
+
+        final List<String> exceptionMessage = verifyCommonInputs(proxyPort, trustAllRoots,
+                connectTimeout, socketTimeout, keepAlive, connectionsMaxPerRoute, connectionsMaxTotal);
+        if (!exceptionMessage.isEmpty()) {
+            return getFailureResultsMap(StringUtilities.join(exceptionMessage, NEW_LINE));
+        }
+
+        final List<String> exceptionMessages = verifyApplyRunRequestBody(requestBody);
+        if (!exceptionMessages.isEmpty()) {
+            return getFailureResultsMap(StringUtilities.join(exceptionMessages, NEW_LINE));
+        }
+
```

Comment:
You can use a bigger function that include both verification so you won't need to create two lists.